ietf/doc/views_doc.py

```diff
@@ -329,6 +329,32 @@ def document_main(request, name, rev=None, document_html=False):
                     css = Path(finders.find("ietf/css/document_html_inline.css")).read_text()
                     if html:
                         css += Path(finders.find("ietf/css/document_html_txt.css")).read_text()
+        # submission
+        submission = ""
+        if group is None:
+            submission = "unknown"
+        elif group.type_id == "individ":
+            submission = "individual"
+        elif group.type_id == "area" and doc.stream_id == "ietf":
+            submission = "individual in %s area" % group.acronym
+        else:
+            if group.features.acts_like_wg and not group.type_id == "edwg":
+                submission = "%s %s" % (group.acronym, group.type)
+            else:
+                submission = group.acronym
+            submission = '<a href="%s">%s</a>' % (group.about_url(), submission)
```

Comment:
We ought to get in the habit of using `mark_safe()` / `format_html()` when we create safe strings instead of `| safe` in the templates.

Reply:
Fair - this is copypasta (essentially) - the change would be good to apply in a comprehensive pass per view.